src/persistence.rs

```diff
@@ -112,11 +106,6 @@ impl<L: Deref + Clone + Send + Sync + 'static> GossipPersister<L> where L::Targe
 				latest_persistence_log = Instant::now();
```

Comment:
Line above `log_info!(self.logger, "Persisting gossip message #{}", i);`

This wasn't accurate, and is may even less accurate now, because persisting happens elsewhere?